commons/src/test/java/org/exoplatform/platform/qa/ui/commons/LoginTestIT.java

```diff
@@ -78,6 +79,7 @@ public void signInWithUnknownUser() {
    */
   @Test
   @Tag("smoke")
+  @Disabled
```

Comment:
Why is it disabled ? Cannot we remove it if it is not used ?

Reply:
It is a test for signOut() and this class extends from Base with contains @afterEach signOut(), so il fails, we remove it ?

Reply:
You can probably move this test in a dedicated test class (for example LogoutTestIT) and override the AfterEach method to not do anything.

Reply:
done